public/stylesheets/main.css

```diff
@@ -1192,6 +1192,50 @@ kbd {
     vertical-align: middle;
 }
 
+.feedback-popover {
```

Comment:
All of this CSS might be necessary, but I'd just ask you to double check that it's all necessary! There have been a lot of iterations, so I just want to make sure that some CSS ended up in here on accident because you were trying stuff out, or it was needed for an older version.

Reply:
That CSS is necessary since that was how I could customize the popovers for specifically /labelMap, /audit, and /validate. The customizations include change of color and position. It also ensures that the popovers don't show up behind the modals :)